types/office-js-preview/index.d.ts

```diff
@@ -281,12 +268,12 @@ declare namespace Office {
      *
      * *Supported applications, by platform*
      *  <table>
-     *   <tr><th>                             </th><th> Office on Windows </th><th> Office in web browser </th><th> Office on iPad </th><th> Outlook on mobile devices </th><th> Office on Mac </th></tr>
-     *   <tr><td><strong> Excel      </strong></td><td> Supported         </td><td> Supported             </td><td> Supported      </td><td>                           </td><td> Supported     </td></tr>
-     *   <tr><td><strong> Outlook    </strong></td><td> Supported         </td><td> Supported             </td><td>                </td><td> Supported                 </td><td> Supported     </td></tr>
-     *   <tr><td><strong> PowerPoint </strong></td><td> Supported         </td><td> Supported             </td><td> Supported      </td><td>                           </td><td> Supported     </td></tr>
-     *   <tr><td><strong> Project    </strong></td><td> Supported         </td><td>                       </td><td>                </td><td>                           </td><td> Supported     </td></tr>
-     *   <tr><td><strong> Word       </strong></td><td> Supported         </td><td> Supported             </td><td> Supported      </td><td>                           </td><td> Supported     </td></tr>
+     *   <tr><th>                             </th><th> Office on the web </th><th> Office on Windows </th><th> Office on Mac </th><th> Office on iPad </th><th> Outlook on mobile devices </th></tr>
+     *   <tr><td><strong> Excel      </strong></td><td> Supported         </td><td> Supported         </td><td> Supported     </td><td> Supported      </td><td> Not applicable            </td></tr>
+     *   <tr><td><strong> Outlook    </strong></td><td> Supported         </td><td> Supported         </td><td> Supported     </td><td> Supported      </td><td> Supported                 </td></tr>
+     *   <tr><td><strong> PowerPoint </strong></td><td> Supported         </td><td> Supported         </td><td> Supported     </td><td> Supported      </td><td> Not applicable            </td></tr>
+     *   <tr><td><strong> Project    </strong></td><td> Not supported     </td><td> Supported         </td><td> Supported     </td><td> Not supported  </td><td> Not applicable            </td></tr>
+     *   <tr><td><strong> Word       </strong></td><td> Supported         </td><td> Supported         </td><td> Supported     </td><td> Supported      </td><td> Not applicable            </td></tr>
```

Comment:
This seems to show the same support matrix as the context one above. Why keep this table?

Reply:
@AlexJerabek I kept this table because it actually gets displayed in the ref docs. I moved the `var context` table to the Context object. I think they so happen to be the same but I don't know that they necessarily must be the same.